src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/fhir/mapper/TransactionMapperConfig.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.fhir.mapper;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.TransactionType;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Configuration
+public class TransactionMapperConfig {
```

Comment:
Just do double check. We are going to use it somewhere, right? Because for now the IntelliJ says to me that we never use that class, possibly it's just my IntelliJ problem.

Reply:
Spring looks at this to create the `Map<TransactionType, FhirTransactionMapper>` and injects it into `EdifactToFhirService`

Reply:
@andrewwa-kainos I see. Thanks for explaining!